doc/development.md

```diff
@@ -28,32 +21,63 @@ Install additional packages for Fedora, CentOS, and RHEL:
 
     sudo dnf install -y `cat automation/check-patch.packages`
 
-Create virtual environment for vdsm:
+Fork the project on https://github.com/oVirt/vdsm.
+
+Clone your fork:
+
+    sudo dnf install -y git
+    git clone git@github.com:{user}/vdsm.git
+
+Generate the Makefile (and configure script):
+
+    ./autogen.sh --system --enable-timestamp
+
+Now you can create the virtual environment
+(https://docs.python.org/3/library/venv.html), which is necessary to run the
+tests later. This needs to be done only once:
 
     make venv
 
 
 ## Building Vdsm
 
-To configure sources (run `./configure --help` to see configuration options):
+Before building, it is recommended to recreate the Makefile because it
```

Comment:
Isn't this an issue only when building RPMs? If so maybe we can mention it in the RPM section.

Reply:
This section is about building rpms, what else do we build?

Reply:
> what else do we build?

Build-time generated py modules needed for tests etc. The RPM part is mentioned only in line 54 of this PR. For `make; tox -e storage` cycles we don't need to run autogen each time.

Reply:
But this section is about building and installing rpms. We have a section about running
the tests. Maybe the heading should be renamed to make it more clear.

Normally make is not needed for running the tests, it is used once when preparing
the source and you need to invoke make only if you change the makefiles or the templates
generated sources. I guess the "running the tests" section need to explain about this.